Code behind --with-relative-data-dir does not do what we want
(which is to use a directory in the home directory), let's not use it.

Index: browser/config/mozconfigs/tor-browser
--- browser/config/mozconfigs/tor-browser.orig
+++ browser/config/mozconfigs/tor-browser
@@ -7,7 +7,7 @@
 # like Firefox does.
 # export MOZ_APP_REMOTINGNAME="Tor Browser"
 
-ac_add_options --with-relative-data-dir=TorBrowser/Data/Browser
+#ac_add_options --with-relative-data-dir=TorBrowser/Data/Browser
 # This is not needed with the relative data directory, but it is when you
 # disable it, otherwise it is "Mozilla" (on Windows and macOS) or ".mozilla" on
 # Unix systems.
